app/controllers/sessions_controller.rb

```diff
@@ -5,16 +5,17 @@ class SessionsController < ApplicationController
   layout 'pages'
 
   def new
-    return create if params[:uid]
+    return create if params.dig('request.omniauth', :uid)
     return redirect_to root_path if current_user
 
     @user = User.new
   end
 
   def create
-    @user = User.find_by email: params[:email]
+    @result = true
```

Comment:
This doesn't feel clean.

Reply:
Just noticed this is still marked WIP.


![](https://c.tenor.com/bUa1N-13OecAAAAC/baby-run-omg.gif)